subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -25,7 +25,6 @@ import io.r2dbc.postgresql.codec.Json
 import io.r2dbc.spi.Row
 import org.slf4j.LoggerFactory
 import org.springframework.data.r2dbc.core.R2dbcEntityTemplate
-import org.springframework.data.r2dbc.core.bind
```

Comment:
hmm, not sure of this.

Reply:
well, actually, there should not be any modification in subscription service in this PR

Reply:
don't forget this